go.mod

```diff
@@ -25,4 +26,6 @@ require (
 	gopkg.in/yaml.v2 v2.2.7
 )
 
+replace github.com/edgexfoundry/go-mod-bootstrap => ../go-mod-bootstrap
```

Comment:
Intentional?

Reply:
Yes, It is to be replaced once go-mod-bootstrap has a tagged version I can use. It will be replaced before I make this a non-draft PR